js_modules/dagster-ui/packages/ui-core/src/instigation/TickMaterializationsTable.tsx

```diff
@@ -0,0 +1,239 @@
+              />
+            );
+          })}
+        </Inner>
+      </div>
+    );
+  };
+
+  return (
+    <Box style={{height: '500px'}} flex={{direction: 'column'}}>
+      <Box
+        padding={{vertical: 12, horizontal: 24}}
+        flex={{justifyContent: 'space-between', alignItems: 'center'}}
+        border="bottom"
+      >
+        <Subtitle2>Requested materializations</Subtitle2>
+        <TextInput
+          icon="search"
+          value={queryString}
+          onChange={(e) => setQueryString(e.target.value)}
+          placeholder="Filter by asset key…"
+          style={{width: '252px'}}
+        />
+      </Box>
+      {content()}
+    </Box>
+  );
+};
+
+const AssetDetailRow = ({
+  $start,
+  $height,
+  assetKey,
+  partitionKeys,
+  evaluationId,
+}: {
+  $start: number;
+  $height: number;
+  assetKey: AssetKeyInput;
+  partitionKeys?: string[];
+  evaluationId: number;
+}) => {
+  const numMaterializations = partitionKeys?.length || 1;
+  const queryResult = useQuery<AssetGroupAndLocationQuery, AssetGroupAndLocationQueryVariables>(
+    ASSET_GROUP_QUERY,
+    {
+      fetchPolicy: 'cache-and-network',
+      variables: {
+        assetKey: {path: assetKey.path},
+      },
+    },
+  );
+  const {data} = queryResult;
+
+  const asset = data?.assetOrError.__typename === 'Asset' ? data.assetOrError : null;
+  const definition = asset?.definition;
+  const repoAddress = definition
+    ? buildRepoAddress(definition.repository.name, definition.repository.location.name)
+    : null;
+  return (
+    <Row $start={$start} $height={$height}>
+      <RowGrid border="bottom">
+        <RowCell>
+          <AssetLink path={assetKey.path} icon="asset" textStyle="middle-truncate" />
+        </RowCell>
+        <RowCell>
+          {data ? (
+            definition && definition.groupName && repoAddress ? (
+              <Link
+                to={workspacePathFromAddress(repoAddress, `/asset-groups/${definition.groupName}`)}
+              >
+                <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+                  <Icon color={Colors.textLight()} name="asset_group" />
+                  {definition.groupName}
+                </Box>
+              </Link>
+            ) : (
+              <Caption color={Colors.textLight()}>Asset not found</Caption>
+            )
+          ) : (
+            <Spinner purpose="body-text" />
+          )}
+        </RowCell>
+        <RowCell>
+          <Link
+            to={assetDetailsPathForKey(assetKey, {
+              view: 'automation',
+              evaluation: `${evaluationId}`,
+            })}
+          >
+            {numMaterializations} materialization{numMaterializations === 1 ? '' : 's'} requested
+          </Link>
+        </RowCell>
+      </RowGrid>
+    </Row>
+  );
+};
+
+const RowGrid = styled(Box)`
+  display: grid;
+  grid-template-columns: ${TEMPLATE_COLUMNS};
+  height: 100%;
+  > * {
+    justify-content: center;
+  }
```

Comment:
What is the `height: 100%` for?

Reply:
I'm actually not totally certain, but its on the `RowGrid` for all of our virtualized table rows so I might leave it to look at separately. Maybe it's necessary to make the cells fill the height allotted to them by the virtualizer?